app/controllers/matchings_controller.rb

```diff
@@ -54,6 +58,111 @@ def destroy
     end
   end
 
+  def category
+    return "person_to_time"
+  end
+
+  def global_settings
+    return {"minutes": @project.duration}
+  end
+
+  def people
+    all_participants_emails = Participant.where(project_id: @project.id).pluck(:email)
+
+    people = []
+    for email in all_participants_emails
+      row = {"name": email, "match_degree": 1}
+      people.push(row)
+    end
+
+    return people
+  end
+
+  def timeslots
+    all_project_times = ProjectTime.where(project_id: @project.id).pluck(:date_time)
+
+    timeslots = []
+    for timeslot in all_project_times
+      timeslot_formatted = timeslot.utc.strftime('%Y-%m-%d %H:%M %p')
```

Comment:
I figured out the the %p was what was causing the API request to fail (as it didn't expect 'AM' or 'PM' in the string), I've gotten it to work now! Will push in a bit.